NewBank/newbank/server/Account.java

```diff
@@ -13,5 +13,16 @@ public Account(String accountName, double openingBalance) {
 	public String toString() {
 		return (accountName + ": " + openingBalance);
 	}
-
+    public double getBalance() {
+		return (openingBalance);
+	}
+	public String getName() {
+		return this.accountName;
+	}
+	public void addMoney(double amount) {
+		this.openingBalance += amount;
+	}
+	public void removeMoney(double amount) {
+		this.openingBalance -= amount;
```

Comment:
Should we check if the account has enough money and return false if it does not? Or is it work for another ticket?

Reply:
Good point; for the implement PAY ticket, checking that the paying account has enough money is part of the ticket. Perhaps we could create a function to check if the account has enough money, that we can then use in both MOVE and PAY?

Reply:
<img width="401" alt="Screen Shot 2021-03-14 at 16 23 01" src="https://user-images.githubusercontent.com/80157394/111075818-9f4c0700-84e1-11eb-890c-1d7000a572e8.png">

Thanks for reviewing this! I've added the check about whether the from account has enough money into the move function itself. Maybe it should be a future ticket if we want to move this to a separate function so it can be used for the pay function too?

Reply:
Sounds good to me :)